rbac/management/audit_log/view.py

```diff
@@ -33,7 +33,13 @@ class AuditLogViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
     serializer_class = AuditLogSerializer
     permission_classes = (AuditLogAccessPermission,)
 
+    def filter_by_id(self, queryset):
+        """Filter queryset by id."""
+        return queryset.order_by("-id").values()
```

Comment:
I am not sure if I understand what is the intention here ... by the method name and docstring it looks like something is supposed to be filtered but the queryset is sorted